test/e2e/gardener/shoot/internal/node/node.go

```diff
@@ -183,11 +183,11 @@ func patchCSINodeObjectWithRequiredDriver(ctx context.Context, shootClient clien
 		},
 	}
 
-	Expect(shootClient.Update(ctx, csiNode)).To(Succeed())
+	ExpectWithOffset(2, shootClient.Update(ctx, csiNode)).To(Succeed())
```

Comment:
Why 2? Also, what about above expectations (L175-176)?

Reply:
It's inside another nested function

Reply:
Yes, and it can be nested again and again. Maybe I'm wrong, but inside such a function, one should probably not make any assumptions about how deeply its invocation is nested? 👀